build.zig

```diff
@@ -3,6 +3,9 @@ const Builder = @import("std").build.Builder;
 pub fn build(b: *Builder) void {
     const target = b.standardTargetOptions(.{});
     const optimize = b.standardOptimizeOption(.{});
+
+    _ = b.addModule("zig-rlp", Builder.CreateModuleOptions{ .source_file = .{ .path = "src/main.zig" } });
```

Comment:
Maybe instead of `zig-rlp` we can name the module as `rlp`, so external projects can use it as: `const mod_rlp = b.dependency("zig-rlp", .{}).module("rlp");` instead of `const mod_rlp = b.dependency("zig-rlp", .{}).module("zig-rlp");`

No strong opinions, so open to comments.